packages/office-ui-fabric-react/src/components/Calendar/Calendar.tsx

```diff
@@ -41,6 +41,7 @@ export class Calendar extends BaseComponent<ICalendarProps, ICalendarState> impl
     [key: string]: React.ReactInstance;
     root: HTMLElement;
```

Comment:
Seems reasonable. @lorejoh12  Can you include a rush change file? (Run `npm run change` from the root. It will compare your branch with origin/master. If you have officedev set up as a separate remote, upstream for example, you can use the parameter `-- -b upstream/master`.)